### PR TITLE
Blank methods and settlement rendering

### PR DESCRIPTION
**Pull request by Evan-Kreutzwiser**

### CODE REVIEW

File: KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java [hunk 1]
```diff
@@ -251,20 +267,6 @@ private void draw(Graphics g) {
             }
         }
 
-        //draw testing art
-        //cards
-        g2d.drawImage(CARD_CLAY, 100, 1080 - 125, null); //space them 100 pixels apart and align the hight to 2 from the bottom
-        g2d.drawImage(CARD_ORE, 200, 1080 - 125, null);
-        g2d.drawImage(CARD_WHEAT, 300, 1080 - 125, null);
-        g2d.drawImage(CARD_WOOD, 400, 1080 - 125, null);
-        g2d.drawImage(CARD_SHEEP, 500, 1080 - 125, null);
```
Comment (Tacitor) [inline KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java:260]: Hey, why kill the cards?

Comment (Evan-Kreutzwiser) [reply KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java:260]: oh sorry i meant to delete the settlement test

Comment (Evan-Kreutzwiser) [reply KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java:260]: Those will be added back in the form of drawing the player's hand/deck anyway

**Review by Tacitor — CHANGES_REQUESTED** [review]
Just bring those placeholder cards back

File: KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java (hunk 1 above)
Comment (Tacitor) [reply KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java:260]: fair enough

**Review by Tacitor — APPROVED** [review]
All good now

**Pull request merged**